docs/contents/components/layouts/divider/index.ja.mdx

```diff
@@ -23,23 +23,25 @@ import { Divider } from "@yamada-ui/react"
 
 ```tsx
 <Flex direction="column" gap="md">
-  <Divider variant="solid" />
-
-  <Divider variant="dashed" />
-
-  <Divider variant="dotted" />
+  <For
+    each={[{ variant: "solid" }, { variant: "dashed" }, { variant: "dotted" }]}
+  >
+    {({ variant }, index) => <Divider key={index} variant={variant} />}
+  </For>
 </Flex>
 ```
 
 ### 垂直に区切り線を引く
 
 ```tsx
 <Flex h="sm" gap="md">
-  <Divider orientation="vertical" variant="solid" />
-
-  <Divider orientation="vertical" variant="dashed" />
-
-  <Divider orientation="vertical" variant="dotted" />
+  <For
+    each={[{ variant: "solid" }, { variant: "dashed" }, { variant: "dotted" }]}
+  >
+    {({ variant }, index) => (
+      <Divider key={index} orientation="vertical" variant={variant} />
+    )}
+  </For>
```

Comment:
This is probably better for readability if we are only looping variant?
```jsx
<For each={["solid", "dashed", "dotted"]}>
  {(variant, index) => (
    <Divider key={index} orientation="vertical" variant={variant} />
  )}
</For>
```

Reply:
Thank you for the great idea! @nathanlao 
I think it would be better to modify the document to ensure consistency in the code style. What do you think? @hirotomoyamada

Reply:
@kaehehehe @nathanlao 

```
<For each={["solid", "dashed", "dotted"]}>
  {(variant, index) => (
    <Divider key={index} orientation="vertical" variant={variant} />
  )}
</For>
```

Please implement with this code.